src/action-sheets/index.js

```diff
@@ -698,6 +716,9 @@ export const constructMessageActionButtons = (args: {|
   if (!isTopicNarrow(narrow) && !isPmNarrow(narrow)) {
     buttons.push(reply);
   }
+  if (canStartQuoteAndReply) {
+    buttons.push(quoteAndReply);
```

Comment:
Probably this shouldn't appear for outbox messages, either. (Fetching their raw content from the server won't work; and they don't yet have a real message ID to link to.)

Reply:
It looks like that's taken care of by the early return at the top of `constructMessageActionButtons`:

```js
  if (message.isOutbox === true) {
    buttons.push(copyToClipboard);
    buttons.push(shareMessage);
    buttons.push(deleteMessage);
    buttons.push(cancel);
    return buttons;
  }
```

Reply:
Ah, great then.